indico/modules/events/persons/forms.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import session, request
+from wtforms.fields import StringField, TextAreaField, SelectField, HiddenField
+from wtforms.validators import DataRequired
+
+from indico.util.i18n import _
+from indico.web.forms.base import IndicoForm
+from indico.web.forms.fields import IndicoStaticTextField, HiddenFieldList
+from indico.web.forms.widgets import CKEditorWidget
+
+
+class EmailEventPersonsForm(IndicoForm):
+    from_address = SelectField(_('From'), [DataRequired()], choices=[(1, 1)])
```

Comment:
why those weird choices? when initializing the choices for a SelectField in `__init__` or afterwards no choices should be provided at all in the field constructor